Library/Formula/entr.rb

```diff
@@ -13,10 +11,19 @@ class Entr < Formula
   end
 
   def install
-    ENV['PREFIX'] = prefix
-    ENV['MANPREFIX'] = man
+    ENV["PREFIX"] = prefix
+    ENV["MANPREFIX"] = man
     system "./configure"
     system "make"
     system "make install"
   end
+
+  test do
+    touch testpath/"test.1"
+    fork do
+      sleep 0.5
+      touch testpath/"test.2"
+    end
```

Comment:
This is a program to monitor file changing. So fork a subprocess to change the file.